apps/challenges/views.py

```diff
@@ -3229,6 +3229,36 @@ def create_or_update_github_challenge(request, challenge_host_team_pk):
                         }
                         send_slack_notification(message=message)
 
+                    template_data = get_challenge_template_data(zip_config.challenge)
+                    if (
+                        not challenge.is_docker_based
+                        and challenge.inform_hosts
```

Comment:
@afif-fahim this check is only neede when sending emails. if `inform_hosts` is false we will start the workers but we will not send any email to update the challenge host. This check should go on line `3251`

Reply:
@Ram81 made the change and updated the PR.